main.go

```diff
@@ -62,6 +62,8 @@ func cmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			context.InferVersion = inferVersion
```

Comment:
Why do we need `inferVersion`? Can we not reference `context.InferVersion` directly?